ingest/a_vs_b_de.py

```diff
@@ -0,0 +1,65 @@
+#using https://nbviewer.org/github/theislab/diffxpy_tutorials/blob/master/diffxpy_tutorials/test/multiple_tests_per_gene.ipynb
+import anndata
+import matplotlib.pyplot as plt
+import seaborn as sns
+import logging
+import numpy as np
+import pandas as pd
+import scipy.stats
+import scanpy as sc
+import os
+
+
+#install with pip install batchglm -- NOT using github repo
+import batchglm.api as glm
+
+#install with pip install diffxpy -- NOT using github repo
+import diffxpy.api as de
+
+#check that batchglm and diffpy were installed correctly
+#print("batchglm version "+glm.__version__)
+#print("diffpy version "+de.__version__)
+
+#Pairwise tests between groups
+#answers whether a given pair of groups shows differential expression for each gene
+
+#arguments:
+#data: Anndata object, data matrix with cells x genes
+
+#grouping: str, column in data, the column that contains cell type labels. alternatively vector containing group labels
+
+#test: type of test, default is z-test. other options: ’wald’, ’lrt’, ’t-test’, ’rank’
+
+#lazy: bool, only possible if test is ztest, if true only evaluated once the user requests pval/coefficients for a specific pair of models
+
+#noise_model: default is nb, specify NONE for wald and t test
```

Comment:
I don't think it's important to do so now, but we'll presumably want to extract these hard-coded arguments into CLI arguments that can be specified at runtime.  

Let's ensure these to CLI arguments is captured in a downstream ticket.  

After that ticket's created, can you add a TODO comment like the following?

```
# TODO (SCP-<your ticket #>): Extract these to CLI arguments
```